fast/stages-multitenant/0-bootstrap-tenant/variables.tf

```diff
@@ -26,11 +26,12 @@ variable "automation" {
     project_number          = string
     federated_identity_pool = string
     federated_identity_providers = map(object({
-      issuer           = string
-      issuer_uri       = string
-      name             = string
-      principal_tpl    = string
-      principalset_tpl = string
+      allowed_audiences = list(string)
```

Comment:
Why are you making `allowed_audiences` a list if you only use a single value?

Reply:
The reason I went with this approach was that the Terraform resource `google_iam_workload_identity_pool_provider` requires a list of `allowed_audiences` but for GitLab it was required only one. However, did some digging and found that's not the case and we can have multiple audiences in Gitlab ID token as well. I've pushed new code to reflect this.

Reply:
much cleaner now. Thanks